### PR TITLE
FPC Rule Update

### PR DESCRIPTION
**Pull request by vvikramb**
FPC Rule Update

### CODE REVIEW

File: juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule [hunk 1]
```diff
@@ -206,18 +206,16 @@ healthbot {
                 description "Sets health based on increase in fpc buffer memory utilization";
```
[the remark this answers is not in the document]
Comment (vvikramb) [reply juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule:206]: Resolved errors

File: juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule [hunk 2]
```diff
@@ -348,65 +360,70 @@ healthbot {
...
                 }
                 /*
-                 * Sets color to green when state is online.
+                 * Sets color to red when state is not online.
                  */				
-                term is-online {
+                term is-not-online {
                     when {
-                        matches-with "$state" online {
+                        does-not-match-with "$state" online {
```
Comment (gaddevijay) [inline juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule:416]: u don't require this when

File: juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule [hunk 3]
```diff
@@ -253,39 +251,50 @@ healthbot {
...
                 /*
                  * Sets color to yellow when fpc-cpu-utilization is greater-than-or-equal-to cpu-low-threshold.
                  */				
                 term is-fpc-cpu-utilization-median {
+                    when {
+                        less-than "$fpc-cpu-utilization" "$cpu-high-threshold" {
+                            time-range 3offset;
+                        }
+                    }				
                     when {
```
Comment (gaddevijay) [inline juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule:298]: 2 when's ????

**Review by gaddevijay — CHANGES_REQUESTED** [review]

File: juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule (hunk 3 above)
Comment (vvikramb) [reply juniper_official/linecard/check-fpc-cpu-memory-state-temp.rule]: Removed 1 when.

**Pull request closed without merging**